msupply-foundation/open-msupply
# 2859 prevent IS changing from new message when status not new

Fixes #2859 

# 👩🏻‍💻 What does this PR do? 
 

<img width="466" alt="Screenshot 2024-02-05 at 12 45 46 PM" src="https://github.com/msupply-foundation/open-msupply/assets/55115239/23aba225-c18a-4cf7-a886-7ddd8cb6f6aa">


# 🧪 How has/should this change been tested? 


1. Go to 'Inbound Shipment'
1. Create a new shipment
1. Add at least 1 item and click on 'Confirm delivered'
1. TEST: See the "Note: changing from 'New'..." text 
1. Click on 'Confirm Verified'
1. TEST: Only see the "Confirm status as Verified" message

## 💌 Any notes for the reviewer?


I updated the translation values for the non-english languages, seeing as it's just separating the existing text across two translations - however, what is the process for updating the other translation files after adding new strings? I saw something in a Readme about Weblate - do I need to create an account there?

## 📃 Documentation


N/A
